homeassistant/components/apsystemsapi_local/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "domain": "apsystemsapi_local",
+  "name": "APsystems Local API",
+  "codeowners": ["@mawoka-myblock", "@SonnenladenGmbH"],
```

Comment:
Just to double check, is Sonnenladen in the loop that they are added as a maintainer of this integration ?

Reply:
@SonnenladenGmbH ?